src/index.html

```diff
@@ -9,10 +9,67 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
+    />
+    <link
+      rel="stylesheet"
+      href="./styles/index.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="imac-photo"
+      />
+      <div>
+        <p class="product-card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+      </div>
+      <p class="product-card__code">Product code: 195434</p>
+
+      <div class="product-card__ratings">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="product-card__review"><p>Reviews: 5</p></div>
+      </div>
+
+      <div class="product-card__prices">
+        <p class="product-card__price">Price:</p>
+        <p class="product-card__dollar-price">$2,199</p>
+      </div>
+
+      <div class="button">
+        <button
+          class="button__buy"
+          data-qa="hover"
+        >
+          BUY
+        </button>
```

Comment:
and here

Reply:
good point !
But in that case it would not follow the methodology by which we write the code, because then the element isn't inside the parent block.